controllers/package_controller_test.go

```diff
@@ -6,11 +6,11 @@ import (
 	"testing"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
Why are we switching out loggers?

Reply:
> Why are we switching out loggers?

We aren't actually switching out loggers. One of the packages upgraded was the logr package. In the new version, they've moved `logr.Logger` from an interface to a struct. As a result, we can't pass around `nil` when we don't care about a logger. As a result, I can't use `nil` in these tests anymore, I have to use `logr.Discard()` instead. So no we're having to import that package in these tests, where before we didn't.